content/spin/manifest-reference.md

```diff
@@ -0,0 +1,140 @@
+title = "Spin Application Manifest Reference"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+[extra]
+url = "https://github.com/fermyon/spin/blob/main/docs/content/manifest-reference.md"
+
+---
+- [Format](#format)
+- [Manifest Fields](#manifest-fields)
+- [The `trigger` Table](#the-trigger-table)
+  - [The `trigger` Table for HTTP Applications](#the-trigger-table-for-http-applications)
+  - [The `trigger` Table for Redis Applications](#the-trigger-table-for-redis-applications)
+- [The `variables` Table](#the-variables-table)
+- [The `component` Tables](#the-component-tables)
+  - [The `component.trigger` Table for HTTP Applications](#the-componenttrigger-table-for-http-applications)
+  - [The `component.trigger` Table for Redis Applications](#the-componenttrigger-table-for-redis-applications)
+  - [The `component.build` Table](#the-componentbuild-table)
+- [Next Steps](#next-steps)
+
+This page describes the contents of the Spin manifest file, typically called `spin.toml`.
+
+## Format
+
+The manifest is a TOML file, and follows standard TOML syntax.  See the [TOML documentation](https://toml.io/) for information about the TOML syntax.
+
+## Manifest Fields
+
+| Name                    | Required?  | Value    | Example   |
```

Comment:
I'm typically mostly confused about (aka cannot remember) what type of data the individual field takes. I think it would be good to add that to the table. I know it hurst with the width, but we have another issue to hopefully fix that :-)

Reply:
Heh, I had thought about that but punted.  Will do!